src/main/resources/se/diabol/jenkins/pipeline/PipelineProperty/help-descriptionTemplate.jelly

```diff
@@ -0,0 +1,12 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <j:new var="h" className="hudson.Functions"/>
+    <!-- instead of JSP functions -->
+    ${h.initPageVariables(context)}
+    <div>
+        <img src="${resURL}/plugin/delivery-pipeline-plugin/help/description.png"/>
+        <p>
+            In a build Delivery Pipeline View this description will be shown as an description for the task. Description can be customized with all environment and <a href="http://ci.jenkins-ci.org/env-vars.html">build variables</a> using <a href="https://wiki.jenkins-ci.org/display/JENKINS/Token+Macro+Plugin">Token Macro Plugin</a>
+        </p>
```

Comment:
Add for example this:
<help xmlns="/lib/token-macro"/>
It will be replaced with a list of all macros available.